apps/admin/pages/arts/collections/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react'
+
+import { MenuItem } from '@chakra-ui/react'
+import { StrapiLocale, Sort } from '@wsvvrijheid/types'
+import { useAuth, AdminLayout, CollectionsTable } from '@wsvvrijheid/ui'
+import { useCollectionsByFilterAndSort } from '@wsvvrijheid/utils'
+import { FaArrowDown, FaArrowUp } from 'react-icons/fa'
+import { useUpdateEffect } from 'react-use'
+
+const CollectionsPage = () => {
+  const { user, isLoading } = useAuth()
+  const [currentPage, setCurrentPage] = useState<number>()
+  // Client side query params (?status=pending)
+  const defaultLocale: StrapiLocale = 'tr'
+
+  const [searchTerm, setSearchTerm] = useState<string>()
+  const [locale, setLocale] = useState<StrapiLocale>(undefined)
+
+  const [sort, setSort] = useState<Sort>()
+  const queryKey = ['collections', searchTerm, sort, currentPage || 1]
+
+  const CollectionsQuery = useCollectionsByFilterAndSort(queryKey, {
+    sort,
+    searchTerm,
+    page: currentPage || 1,
+    locale: locale as StrapiLocale,
+  })
+  const handleSearch = (search: string) => {
+    search ? setSearchTerm(search) : setSearchTerm(undefined)
+  }
+
+  useUpdateEffect(() => {
+    CollectionsQuery.refetch()
+  }, [locale, searchTerm, sort])
+
+  const Collections = CollectionsQuery?.data?.data
```

Comment:
camelCase is preferable for variable names

```suggestion
  const collections = CollectionsQuery?.data?.data
```